### PR TITLE
Added Reviews model/controller and unit tests

### PR DESCRIPTION
**Pull request by joeziemba**
Installed FactoryBot
Defined user, venue, review factories
Added Review migration
Added Review model with associations and validations
Added Reviews Controller (empty for now)
Added unit tests for Reviews (ratings, associations)

### CODE REVIEW

File: app/assets/javascripts/review.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (jdmacmurtrie) [inline app/assets/javascripts/review.coffee]: You don't need this file

File: spec/factories.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    #   after(:create) do |venue, evaluator|
+    #     create_list(
+    #         :review,
+    #         evaluator.review_count,
+    #         venue: venue,
+    #         user: User.first
+    #       )
+    #   end
+    # end
+  end
```
Comment (jdmacmurtrie) [inline spec/factories.rb]: Clean up commented code before submitting a PR please

File: spec/factories.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  factory :review do
+    rating 4
+    user
+    venue
+  end
+end
```
Comment (jdmacmurtrie) [inline spec/factories.rb]: These factories should all be divided into separate files.

**Comment by joeziemba** [comment]
@jdmacmurtrie 
Deleted review.coffee
Split factories.rb into 3 files (one for each: user/venue/review) in factories folder
Added /tmp to .gitignore

**Pull request merged**